src/context.rs

```diff
@@ -284,20 +284,24 @@ pub unsafe fn dc_context_unref(context: &mut Context) {
 }
 
 pub unsafe fn dc_close(context: &Context) {
+    info!(context, 0, "disconnecting INBOX-watch",);
     context.inbox.read().unwrap().disconnect(context);
+    info!(context, 0, "disconnecting sentbox-thread",);
     context
         .sentbox_thread
         .read()
         .unwrap()
         .imap
         .disconnect(context);
+    info!(context, 0, "disconnecting mvbox-thread",);
     context
         .mvbox_thread
         .read()
         .unwrap()
         .imap
         .disconnect(context);
 
+    info!(context, 0, "disconnecting SMTP");
```

Comment:
do we want to keep all these `info` blocks? (I am fine either way)

Reply:
if termination proves to be stable we can remove some ;)